catalyst_voices/packages/internal/catalyst_voices_repositories/lib/src/utils/json_converters.dart

```diff
@@ -31,3 +31,32 @@ final class ShelleyAddressConverter
   @override
   String toJson(ShelleyAddress object) => object.toBech32();
 }
+
+final class ListStringConverter
+    implements JsonConverter<List<String>, List<dynamic>> {
+  const ListStringConverter();
+
+  @override
+  List<String> fromJson(List<dynamic>? json) {
+    if (json == null) return <String>[];
+    return json.map((e) => e as String).toList();
+  }
+
+  @override
+  List<dynamic> toJson(List<String> object) =>
+      object.map((e) => e as dynamic).toList();
+}
```

Comment:
Should be a converter that can work with nullable types. In case of converting a `DocumentProperty.value` that was set to null it will throw a runtime error if nullable types are not supported.

```suggestion
final class ListStringConverter
    implements JsonConverter<List<String>?, List<dynamic>?> {
  const ListStringConverter();

  @override
  List<String>? fromJson(List<dynamic>? json) {
    if (json == null) return null;
    
    return json.cast();
  }

  @override
  List<dynamic>? toJson(List<String>? object) => object;
}
```